masterlist.yaml

```diff
@@ -2472,6 +2472,26 @@ plugins:
         subs: [ 'Low Robots ESP' ]
         condition: 'many("AWOP(Very)?LowRobots\.esp")'
 
+  - name: 'AWorldOfPainFO3.esm'
+    url:
+      - link: 'https://www.nexusmods.com/newvegas/mods/66265'
+        name: 'TTW A World of Pain for Fallout 3'
+    req: [ *TTWv320 ]
+    msg:
+      - <<: *patch3rdParty
+        subs:
+          - 'Literally Who''s Patch Emporium'
+          - '[ AWorldOfPainFO3 - FafnirUniqueTTWOverhaul Patch ](https://www.nexusmods.com/newvegas/mods/83169/)'
```

Comment:
The empty spaces before and after `AWorldOfPainFO3 - FafnirUniqueTTWOverhaul Patch` (so within the brackets `[ ]`) can be removed.